setup.py

```diff
@@ -28,6 +28,7 @@
     'six',
     'transaction',
     'zope.sqlalchemy',
+    'python-ldap',
```

Comment:
I actually preferred what you were doing earlier with `extras_require`. It would be better to not require every installation to install `python-ldap`